posthog/management/commands/test/test_fix_person_distinct_ids_after_delete.py

```diff
@@ -0,0 +1,83 @@
+import logging
+from typing import Dict
+from uuid import uuid4
+
+import pytest
+
+from posthog.client import sync_execute
+from posthog.management.commands.fix_person_distinct_ids_after_delete import run
+from posthog.models.person.person import Person, PersonDistinctId
+from posthog.models.person.sql import PERSON_DISTINCT_ID2_TABLE
+from posthog.models.person.util import create_person, create_person_distinct_id
+from posthog.test.base import BaseTest, ClickhouseTestMixin
+
+
+@pytest.mark.ee
+class TestFixPersonDistinctIdsAfterDelete(BaseTest, ClickhouseTestMixin):
+    def create_test_person(self, distinct_ids: Dict[str, int]):
+        person = Person.objects.create(team_id=self.team.pk, version=0, uuid=uuid4())
+        create_person(uuid=str(person.uuid), team_id=person.team.pk, version=person.version)
+        for did, version in distinct_ids.items():
```

Comment:
nit: everywhere else we use `pdi` as the abbreviation for person distinct id. I personally don't like these abbreviations much